modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/domain/cell/CoalgebraCommand.scala

```diff
@@ -11,4 +13,6 @@ object CoalgebraCommand {
   case class ProcessDAGL1(data: Signed[Block]) extends CoalgebraCommand
   case class ProcessStateChannelSnapshot(snapshot: StateChannelOutput) extends CoalgebraCommand
   case class ProcessUpdateNodeParameters(updateNodeParameters: Signed[UpdateNodeParameters]) extends CoalgebraCommand
+  case class ProcessDelegatedStake(createDelegatedStake: DelegatedStakeOutput) extends CoalgebraCommand
```

Comment:
Same issue as above, this should be a unified type since we're processing them all the same way